src/BundleRegistry.js

```diff
@@ -24,10 +24,28 @@ define(function () {
 
     function BundleRegistry() {
         this.bundles = {};
+        this.knownBundles = {};
     }
 
     BundleRegistry.prototype.register = function (path, definition) {
-        this.bundles[path] = definition;
+        if (this.knownBundles.hasOwnProperty(path)) {
```

Comment:
Ah, presumably this threw up a problem with the duplicate notification path in the MSL module.